.changeset/pre.json

```diff
@@ -15,7 +15,6 @@
     "example-backend-legacy": "0.2.101",
     "@backstage/backend-openapi-utils": "0.1.16",
     "@backstage/backend-plugin-api": "0.8.0",
-    "@backstage/backend-tasks": "0.6.0",
```

Comment:
Is this right?

Reply:
Should be fine to remove if there's nothing left to release from now on 👍